website/pages/en/releases.js

```diff
@@ -66,23 +63,25 @@ function Releases(props) {
               </tr>
             </thead>
             <tbody>
-              {releases.map((release) => (
-                <tr key={release.version}>
-                  <td>{release.version}</td>
-                  <td>
-                    finos/legend-studio:
-                    {release.manifest.core["finos/legend-studio"]}
-                  </td>
-                  <td>
-                    finos/legend-engine-server:
-                    {release.manifest.core["finos/legend-engine-server"]}
-                  </td>
-                  <td>
-                    finos/legend-sdlc-server:
-                    {release.manifest.core["finos/legend-engine-server"]}
-                  </td>
-                </tr>
-              ))}
+              {releases
+                .sort((a, b) => b.timestamp - a.timestamp)
```

Comment:
I see how you are using a timestamp to sort the table.

However, this is extra maintenance overhead. Since we are using a date as our version number, can we just parse the version as date ? (and maybe support a suffix in case we do multiple releases per day).

Alternatively, we can commit to semver for versioning.

Reply:
I don't think we agreed on doing `semver` in the first place. If that's the case in the future, we will probably have to change the whole implementation here, less to say we still need to keep these older versions around for record, so we will have a case of mixed versioning-schemes (date vs. semver), which makes using timestamp for sorting even more sensible. As for the suffix, other than `.2`, `.3` to mark multiple versions released on the same date, the needs for things like `-alpha`, `-beta` might come into play. Thus, it can potentially make the sorting of version here necessarily complicated, again this begs the question why don't we just use the timestamp to sort.

I'm not sure I see a better way than using `timestamp`, hence why I opted for this approach and took the time to go back and add them to the manifest files. I understand what you mean by the `overhead`, but I don't think it's that bad, given that these kinds of work will be automated in the near future anyway and that we don't release that frequently as of now.

Reply:
K. Hopefully we can get the releases automated soon!